website/app/styles/components/copy-button.scss

```diff
@@ -5,9 +5,13 @@
 .doc-copy-button {
   display: inline-flex;
   align-items: center;
-  max-width: 100%;
+  max-width: fit-content;
```

Comment:
@jorytindall I tend to use `fit-content` very rarely, because despite what the name says, it tends to do much more than one would expect (and behave unexpectedly).
I've just merged #735 in `main` and if you pull you will see that with the proposed change, this happens:

**with current code (`max-width: 100%`)**
<img width="457" alt="screenshot_2181" src="https://user-images.githubusercontent.com/686239/207428872-bcee3ae3-dfab-41af-a1cf-e5f652cb0fb9.png">

**with proposed code (`max-width: fit-content`)**
<img width="445" alt="screenshot_2182" src="https://user-images.githubusercontent.com/686239/207429002-4197ff47-5cb7-4689-a90d-4f37ab267d96.png">

(try to change the value in the browser here https://hds-website-hashicorp.vercel.app/foundations/icons/ and see what happens)

Reply:
@MelSumner if you wrap the `Doc::CopyButton` in a container (eg. a `<div>`) instead of using it as direct child of the grid, it will fix the issue. Or don't use a grid at all but a flexbox (see if how it's done here suits your needs: https://hds-website-hashicorp.vercel.app/foundations/icons/)

Reply:
@didoo ok got it, makes sense, thanks for the note